### ISSUE TITLE
A typo

### ISSUE THREAD

**Issue by zenghsh3** [xiaojunxu/SQLNet#9]
https://github.com/xiaojunxu/SQLNet/blob/8997542839ba5815300f088d28bdb6b5f7613136/sqlnet/model/seq2sql.py#L179

It should be len(gt_where[b]).

**Comment by xiaojunxu** [comment]
No, the loss is averaged over all the cases in the batch (the number is len(gt_where)). So it is divided by len(gt_where).

**Comment by zenghsh3** [comment]
Oh, sorry! I got it wrong. nn.CrossEntropyLoss has average the loss of each where tokens.
